keploy.sh

```diff
@@ -33,10 +35,96 @@ installKeploy (){
             ;;
         esac
     done
+
+    # Function to check for macOS version compatibility
+    check_macos_version() {
```

Comment:
Hey @Harshjosh361 We dont require to check mac version for the compatibilty. Also we also have a binary for mac so we don't require to set the alias expliciltly.